app/views/categories/index.html.erb

```diff
@@ -20,6 +20,13 @@
                 </a>
               </h3>
               <div itemprop='description'><%= c.description&.html_safe %></div>
+              <% if c.subcategory_list.present? %>
+                <div class='subcategories'>
+                  <% c.subcategory_list&.each_with_index do |sc, index| %>
```

Comment:
Nitpick, but you probably don't need the `&.` now that we're testing for `.present?`